src/DotRecast.Core/Collections/RcSortedQueue.cs

```diff
@@ -55,7 +55,7 @@ private void Balance()
         {
             if (_dirty)
             {
-                _items.Sort(_comparer); // reverse
+                _items.Sort(_comparison); // reverse
```

Comment:
List<T>.Sort(IComparer<T>) allocates Comparison<T>, so we pass delegate directly to avoid allocations.